website/docs/docs/cloud/git/setup-azure.md

```diff
@@ -5,20 +5,28 @@ description: "You can set up your Azure DevOps by creating a Microsoft Entra ID
 sidebar_label: "Set up Azure DevOps"
 ---
 
-<Snippet path="available-enterprise-tier-only" />
+# Set up Azure DevOps <Lifecycle status="enterprise" />
 
-## Overview
+## Service principal overview
+
+:::note
+
+You can use these instructions to create a service principal app. The service principal feature is being gradually rolled out to dbt Cloud accounts, so it may not be available in all accounts. 
+
+If you need to configure dbt Cloud and Azure DevOps but don't see the **Service principal** option in your account settings, use the [**Service user**](#set-up-service-user) configuration for now. You can easily [migrate to a service principal](#migrate-to-service-principal) when available for your account.
```

Comment:
This URL for service user doesn't work. I think it needs to be `#connect-a-service-user`